### PR TITLE
feat(branding): new branding

### PR DESCRIPTION
**Pull request by ExFlo**

### CODE REVIEW

File: demo/src/routes/+page.svelte [hunk 1]
```diff
@@ -16,81 +15,98 @@
...
+		</LandingCard>
+		<LandingCard title="Bootstrap widgets">
+			Highly customizable Bootstrap CSS widgets are available for Angular, React and Svelte in separated packages.
+		</LandingCard>
+		<LandingCard title="Consistent">Widgets are provided for Angular, React and Svelte with the exact same markup and functionalities.</LandingCard>
+		<LandingCard title="Highly configurable">
+			Default values for all widgets props are configurable at any point of the application subtree.
+		</LandingCard>
+		<LandingCard title="Fully tested">Unit tests for headless with Vitest. E2e tests for adapters with Playwright.</LandingCard>
+		<LandingCard title="Accessible">Widgets are tested for accessibility with Axe.</LandingCard>
```
Comment (andreascorti) [inline demo/src/routes/+page.svelte:103]: Suggestion: last card is alone on the row, shouldn't we center it ?

**Review by andreascorti — COMMENTED** [review]
I still see many headings that are not #26005A but --bs-body-color.
Is it normal? 
And I see other things: 
1) hover effect on menu 
2) Tabs selected color
3) tabs are not working anymore. Not sure it is linked to the PR

File: demo/src/routes/+layout.svelte [hunk 2]
```diff
@@ -51,30 +52,30 @@
 -->
 
 <div class="agnos-ui">
-	<nav class="demo-nav-top navbar navbar-dark bg-primary bg-gradient">
+	<nav class="demo-nav-top navbar">
```
[the remark this answers is not in the document]
Comment (ExFlo) [reply demo/src/routes/+layout.svelte:55]: ok

File: demo/src/routes/+page.svelte (hunk 1 above)
Comment (ExFlo) [reply demo/src/routes/+page.svelte:103]: not yet.

**Review by andreascorti — APPROVED** [review]
LGTM thanks @ExFlo !

**Pull request merged**